beacon-send.html

```diff
@@ -40,89 +40,86 @@
 @status beta
 @homepage https://github.com/ebidel/beacon-send
 -->
-<polymer-element name="beacon-send" attributes="data url dontSendOnClose" hidden>
+
 <script>
-(function() {
-
-var UNLOAD_EVENT_ = 'unload';
-
-Polymer({
-
-  /**
-   * Fired when the message was queued. You should only need this event when
-   * using `send()` explicitly.
-   *
-   * @event beacon-queue
-   * @param {Object} detail
-   *   @param {string} detail.success True if the message was queued successfully.
-   */
-
-  /**
-   * The url of the remote server to send data to.
-   *
-   * @attribute url
-   * @type string
-   * @default null
-   */
-  url: null,
-
-  /**
-   * The data to send to the remove server.
-   *
-   * @attribute data
-   * @type ArrayBufferView|Blob|String|FormData
-   * @default null
-   */
-  data: null,
-
-  /**
-   * If `true`, does not send data when the page's `unload` event is fired.
-   *
-   * @attribute dontSendOnClose
-   * @type bool
-   * @default true
-   */
-  dontSendOnClose: false,
-
-  created: function() {
-    if (!('sendBeacon' in navigator)) {
-      console.warn(this.localName +
-                   ': the Beacon API is not available in this browser.');
-    }
-  },
-
-  ready: function() {
-    this.unloadHandler_ = this.send.bind(this);
-    this.dontSendOnCloseChanged();
-  },
-
-  dontSendOnCloseChanged: function() {
-    if (this.dontSendOnClose) {
-      window.removeEventListener(UNLOAD_EVENT_, this.unloadHandler_);
-    } else {
-      window.addEventListener(UNLOAD_EVENT_, this.unloadHandler_);
+  Polymer({
+    is: 'beacon-send',
+
+    /**
+     * Fired when the message was queued. You should only need this event when
+     * using `send()` explicitly.
+     *
+     * @event beacon-queue
+     * @param {Object} detail
+     *   @param {string} detail.success True if the message was queued successfully.
+     */
+
+    properties: {
+
+      /**
+       * The url of the remote server to send data to.
+       *
+       * @attribute url
```

Comment:
you can remove `@attribute` `@type` and `@default` from the doc strings. They are picked up from the actual code now.